code/modules/hydroponics/hydroitemdefines.dm

```diff
@@ -221,80 +221,79 @@
 // *************************************
 
 
-/obj/item/reagent_containers/glass/bottle/nutrient
-	name = "bottle of nutrient"
+/obj/item/reagent_containers/glass/jug/nutrient
+	name = "jug of nutrient"
 	icon = 'icons/obj/chemical.dmi'
-	icon_state = "bottle16"
-	volume = 50
+	icon_state = "plastic_jug"
 	w_class = WEIGHT_CLASS_TINY
-	amount_per_transfer_from_this = 10
-	possible_transfer_amounts = list(1,2,5,10,15,25,50)
 
-/obj/item/reagent_containers/glass/bottle/nutrient/New()
+/obj/item/reagent_containers/glass/jug/nutrient/New()
```

Comment:
I think you can replace this with `Initialize(mapload)` for this, might want to test it as well. It will be managed by a subsystem which helps with lag and such. `mapload` being the param that says if the jug is created when the map first loads or not

Reply:
Due to essential code being defined using New() in reagent_containers.dm and some call order problems, I will be using New() instead of Initialize() in this PR. Until someone refactors all reagent containers to use Initialize it will have to remain so.